cumulusci/tasks/tests/test_create_package_version.py

```diff
@@ -153,6 +154,15 @@ def mock_get_static_dependencies():
 
 
 class TestPackageConfig:
+    def test_org_config(self, project_config, org_config):
+        org_config.config_file = None
+        with pytest.raises(
+            TaskOptionsError,
+            match=PERSISTANT_ORG_ERROR,
+        ):
+            task = CreatePackageVersion(project_config, TaskConfig(), org_config)
+            task.__init__
```

Comment:
This can be removed:

```suggestion
```

Note that the`__init__` method is the class constructor and is called during initialization.